### ISSUE TITLE
Recalcuate regional records tags if a user's region changes and they have a record tag

### ISSUE THREAD

**Issue by dunkOnIT** [thewca/wca-live#192]
This prevents situations like [this user](https://live.worldcubeassociation.org/competitions/3353/competitors/357439) achieving Yemenese NR, having their region changed to US, and showing up as having achieved a US national record.

![image](https://github.com/thewca/wca-live/assets/52967253/72b728a1-ae54-4e80-ba58-4f099ce96451)

**Comment by jonatanklosko** [comment]
Ideally it would update, although in practice it's very unlikely because it requires the competitor to change country right before or during the competition. And in cases like this it's enough to resubmit the result to force record tags to be recomputed.

**Comment by dunkOnIT** [comment]
Do you suggest we close this issue and just use resubmitting the results as a manual workaround? 

Also, how were you resubmitting the results? Literally just bringing up their scorecard input and submitting it again?

**Comment by jonatanklosko** [comment]
> Do you suggest we close this issue and just use resubmitting the results as a manual workaround?

Yeah, IMO it's not worth the additional complexity, given how rare this is.

> Also, how were you resubmitting the results? Literally just bringing up their scorecard input and submitting it again?

No need for scorecard, you just go to the round editing, select the given competitor and re-submit the results form (it is already filled with the entered times).
